python/pyspark/pandas/plot/core.py

```diff
@@ -857,14 +857,13 @@ def box(self, **kwds):
 
         Parameters
         ----------
-        **kwds : optional
-            Additional keyword arguments are documented in
-            :meth:`pyspark.pandas.Series.plot`.
-
         precision: scalar, default = 0.01
             This argument is used by pandas-on-Spark to compute approximate statistics
             for building a boxplot. Use *smaller* values to get more precise
-            statistics (matplotlib-only).
+            statistics (matplotlib-only).cccccbdvtdlhreffhieutnkglfeibhferhfctieuiiln
```

Comment:
what is this?

Reply:
> matplotlib-only
after this change, will it also take effect in plotly?

Reply:
I wasn’t sure why we added “matplotlib-only” in the first place, because it works with Plotly as shown [here](https://github.com/apache/spark/blob/master/python/pyspark/pandas/plot/plotly.py#L166).
Removed.